### ISSUE TITLE
Make timer tooltip

### ISSUE THREAD

**Issue by stevenjchang** [HashtagCoding/water-clock#3]
Timer - hovering animation - tooltip only

- tooltip (helpful text) on hover
- future - hp bar

Masaya

**Comment by masayafunakoshi1** [comment]
Just to clarify, should I try and make this completely from scratch? Or could I use Material UI's Popover? 
Also, without the actual timer component available, I've just made a dummy timer to put the CSS and JS on.

**Comment by stevenjchang** [comment]
@masayafunakoshi1 

you can do either. Material UI is fine.

would be good if you understood at a high level how Material is doing it, or how to do it from scratch.

The concepts I can think of are:

- css pseudo class
- :hover

(possibly)
- ::before and ::after

---

Dummy timer or any text box is fine for now.

**Comment by masayafunakoshi1** [comment]
Got it, I created a Popover component with Material UI along with CSS styling. Though I'm not too sure I understand it at a "high-level". I think I can explain it and change it around if need be, and added explanations to my understanding. Should I just push it so you can take a look?

**Comment by stevenjchang** [comment]
yeah, go ahead and push your branch

**Comment by masayafunakoshi1** [comment]
Hey Steve, I'm a bit stuck on trying to push the commits I made to my own branch. With git checkout, it says I'm already on "mf/hovering" but when I try and commit then push it, it says "Permission to HashtagCoding/water-clock.git denied to masayafunakoshi1".

Not sure if I'm doing something wrong. I tried figuring it out myself, but still have no clue.

**Comment by stevenjchang** [comment]
I've never seen that msg before but I looked it up,

probably 1 of a few possible things, ultimately will probably be easier to solve on a call. I have some time tomorow.

maybe you have multiple github accounts, or at least multiple ssh keys.

or something else along those lines. https://stackoverflow.com/questions/5335197/gits-famous-error-permission-to-git-denied-to-user

**Comment by masayafunakoshi1** [comment]
Yeah sounds good, I'll keep playing around with it a little more to see if I can get it in. Just was worried that I might mess something up and ruin the project, but since I don't have access to push into the main branch it should be fine right?

Definitely would like to call, I won't be available at 7pm tomorrow unfortunately. I'm free tomorrow afternoon or later that night after 9, whatever works better for you
